guides/mastering_transceivers.md

```diff
@@ -0,0 +1,536 @@
+
+```js
+pc1 = new RTCPeerConnection();
+pc2 = new RTCPeerConnection();
+
+tr1 = pc1.addTransceiver("audio");
+tr2 = pc1.addTransceiver("video");
+
+offer = await pc1.createOffer();
+await pc1.setLocalDescription(offer);
+await pc2.setRemoteDescription(offer);
+answer = await pc2.createAnswer();
+await pc2.setLocalDescription(answer);
+await pc1.setRemoteDescription(answer);
+
+tr1.stop();
+tr2.stop();
+
+offer = await pc1.createOffer();
+await pc1.setLocalDescription(offer);
+await pc2.setRemoteDescription(offer);
+answer = await pc2.createAnswer();
+await pc2.setLocalDescription(answer);
+await pc1.setRemoteDescription(answer);
+
+// negotiate once again even though negotiation is not needed
+offer = await pc1.createOffer();
+await pc1.setLocalDescription(offer);
+
+// observe that after setting remote offer with unused m-lines,
+// stopped transceivers are created...
+await pc2.setRemoteDescription(offer);
+console.log(pc2.getTransceivers());
+
+answer = await pc2.createAnswer();
+// ...and removed 
+await pc2.setLocalDescription(answer);
+console.log(pc2.getTransceivers());
+
+await pc1.setRemoteDescription(answer);
+```
+
+### Elixir WebRTC
+
+```elixir
+{:ok, pc1} = PeerConnection.start_link()
+{:ok, pc2} = PeerConnection.start_link()
+
+{:ok, tr1} = PeerConnection.add_transceiver(pc1, :audio)
+{:ok, tr2} = PeerConnection.add_transceiver(pc1, :video)
+
+{:ok, offer} = PeerConnection.create_offer(pc1)
+:ok = PeerConnection.set_local_description(pc1, offer)
+:ok = PeerConnection.set_remote_description(pc2, offer)
+{:ok, answer} = PeerConnection.create_answer(pc2)
+:ok = PeerConnection.set_local_description(pc2, answer)
+:ok = PeerConnection.set_remote_description(pc1, answer)
+
+:ok = PeerConnection.stop_transceiver(pc1, tr1.id)
+:ok = PeerConnection.stop_transceiver(pc1, tr2.id)
+
+{:ok, offer} = PeerConnection.create_offer(pc1)
+:ok = PeerConnection.set_local_description(pc1, offer)
+:ok = PeerConnection.set_remote_description(pc2, offer)
+{:ok, answer} = PeerConnection.create_answer(pc2)
+:ok = PeerConnection.set_local_description(pc2, answer)
+:ok = PeerConnection.set_remote_description(pc1, answer)
+
+{:ok, offer} = PeerConnection.create_offer(pc1)
+:ok = PeerConnection.set_local_description(pc1, offer)
+:ok = PeerConnection.set_remote_description(pc2, offer)
+
+dbg(PeerConnection.get_transceivers(pc2))
+
+{:ok, answer} = PeerConnection.create_answer(pc2)
+:ok = PeerConnection.set_local_description(pc2, answer)
+
+dbg(PeerConnection.get_transceivers(pc2))
+
+:ok = PeerConnection.set_remote_description(pc1, answer)
+```
+
+<!-- tabs-close -->
+
+## Recycling m-lines
+
+When calling stop on an `RTCRtpTransceiver`, it will eventually be removed from
+the connection's set of transceivers.
+However, the number of m-lines in SDP offer/answer can never decrease.
+`m-lines` corresponding to stopped transceivers can be reused when a new transceiver appears.
```

Comment:
This the only place where m-lines is written using backticks (`m-line`). Make is consistent.